### PR TITLE
Feature/clean up tests

### PR DESCRIPTION
**Pull request by franzejr**

### CODE REVIEW

**Comment by l3x** [comment]
It's ready for a review.

File: db/seeds.rb [hunk 1]
```diff
@@ -68,14 +68,14 @@ def log(msg)
 alice = Receiver.create!(
   receiver_sso_id: Faker::Number.number(digits = 8),
   email: 'alice@aol.com',
-  mobile_number: Faker::PhoneNumber.cell_phone,
+  mobile_number: ENV['ALICES_MOBILE_NUMBER'],
```
Comment (franzejr) [inline db/seeds.rb]: why do you have this as an env var?

File: db/seeds.rb [hunk 2]
```diff
@@ -68,14 +68,14 @@ def log(msg)
...
   first_name: 'Alice',
   last_name: 'Green',
   preferences: {email: false, sms: true}
 )
 bob = Receiver.create!(
   receiver_sso_id: Faker::Number.number(digits = 8),
-  email: 'bob@gmail.com',
+  email: ENV['BOBS_EMAIL'],
```
Comment (franzejr) [inline db/seeds.rb]: why do you have this as an env var?

File: db/seeds.rb (hunk 1 above)
Comment (l3x) [reply db/seeds.rb]: Because I don't want the actual cell phone that I use for testing to be stored in git.

File: db/seeds.rb (hunk 2 above)
Comment (l3x) [reply db/seeds.rb]: Because I don't want the actual email that I use for testing to be stored in git.

**Pull request merged**